### PR TITLE
Video-block-redesign-block-redesign-643

### LINKED ISSUE #643: :art:  :lipstick: Video Block Redesign: block redesign
## User stories

As a user, I want the "Video Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/video-message-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/video-message-block/`.

### PR DESCRIPTION
**Pull request by philipNgungi037**
# Description

The PR redesigns the video block. 
- Changing icons to svg.
- change js plum end-point.
- increase border smoothness.
- remove border outline.

Fixes #643

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
video-block

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/video-block.jsplumb.ts [hunk 1]
```diff
@@ -21,8 +21,8 @@ export function _VideoBlockDecoratePlumb(block: VideoMessageBlock, comp: Compone
     // Whether the anchor is target (Other Block -> This Block)
     target: true,
     cssClass:"block_endpoint",
-    endpoint: "Rectangle",
-    anchor: "Left",
+    endpoint: "Dot",
+    anchor:  [0, 0.11, 0, 1],
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/video-block.jsplumb.ts:25]: have this as "Left"

File: libs/features/convs-mgr/stories/blocks/library/video-message-block/src/lib/components/video-block/video-block.component.html [hunk 2]
```diff
@@ -17,7 +18,6 @@
         </ng-template>
       </div>
     </div>
-    <app-default-option-field [jsPlumb]="jsPlumb" [blockFormGroup]="videoMessageForm"></app-default-option-field>
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/video-message-block/src/lib/components/video-block/video-block.component.html:20]: Add the default option back

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]

**Comment by philipNgungi037** [comment]
I have made changes. @LemmyMwaura

**Comment by LemmyMwaura** [comment]
> I have made changes. @LemmyMwaura

The default option is still missing.

**Comment by LemmyMwaura** [comment]
Closing as it's been resolved

**Pull request closed without merging**